example/README.md

```diff
@@ -0,0 +1,50 @@
+# Example Multichain Deployment
+
+This guide provides a walkthrough for a basic multichain deployment scenario, illustrating how to deploy a sample smart contract across multiple blockchain networks. This example is designed to showcase the versatility and ease of managing deployments in a multichain environment using Hardhat.
+
+## Getting Started
+
+Before diving into the deployment process, you must first set up your environment. This involves adding your private key to the project configuration. Your private key is essential for signing transactions during the deployment process. To set up your environment, execute the following command in your terminal:
+
+```shell
+npx hardhat var set PK
+```
+
+This command securely stores your private key, enabling Hardhat to interact with blockchain networks on your behalf.
```

Comment:
are you sure it's secure? 😂

Reply:
Secure from accidentally commit :)

Reply:
maybe mention that instead, secure storage is misleading :)